inc/checks/class-cache-flush.php

```diff
@@ -36,6 +32,6 @@ public function run() {
 		}
 
 		$this->set_status( 'warning' );
-		$this->set_message( 'Use of wp_cache_flush() detected.' );
+		$this->set_message( 'Use of wp_cache_flush() detected in ' . implode( ', ', $this->_matches ) );
```

Comment:
What do you think about improving the output readability, e.g., by removing the site path from the matches to display relative paths?

Reply:
I have now modified the code to include relative paths only.